api/src/test/java/org/openmrs/aop/RequiredDataAdviceTest.java

```diff
@@ -567,8 +567,8 @@ public void before_shouldCallHandlerOnSaveMethodNameWithCollectionArgument() thr
 		Method m = WithAppropriatelyNamedMethod.class.getMethod("saveSomeOpenmrsDatas", List.class);
 		List<SomeOpenmrsData> openmrsObjects = Arrays.asList(new SomeOpenmrsData(), new SomeOpenmrsData());
 		requiredDataAdvice.before(m, new Object[] { openmrsObjects }, new WithAppropriatelyNamedMethod());
-		verify(saveHandler, times(2)).handle(Matchers.<SomeOpenmrsData> anyObject(), Matchers.<User> anyObject(),
-		    Matchers.<Date> anyObject(), anyString());
+		verify(saveHandler, times(2)).handle(Matchers.<SomeOpenmrsData> anyObject(), Matchers.anyObject(),
```

Comment:
I thought you were also going to get rid of SomeOpenmrsData?

Reply:
Yeah, didn't see that. Fixed 😊